std/digest/crc.d

```diff
@@ -186,11 +188,11 @@ struct CRC32
             this = CRC32.init;
         }
         ///
-        unittest
+        @safe unittest
         {
             CRC32 digest;
             //digest.start(); //Not necessary
-            digest.put(0);
+            digest.put(ubyte(0));
```

Comment:
Does `put(0)` now not work? That seems like a breaking change, especially when it's called out in an example.